internal/scriptrun/scriptrun.go

```diff
@@ -196,6 +198,10 @@ func applySuffix(suffix string, paths []string) []string {
 }
 
 func isExecutableFile(name string) bool {
+	if !fileutils.FileExists(name) {
+		name = executor.NameForExe(name)
+	}
```

Comment:
I'm not a fan of including executor code here. I think the fix for this function should be to check if the ext of `name` is on PATHEXT.

Reply:
In this case `language.Executable().Filename()` will return `perl.exe` so that extension should always be on `PATHEXT`. Since we are using executors we need to search for `perl.bat` or `perl.exe.bat`. We could potentially update the `language` package to return the executor filename for languages that we don't allow 3rd party usage of (ie. if [this](https://github.com/ActiveState/cli/blob/master/internal/language/language.go#L252) field is false).

Reply:
My point is that .bat should be on PATHEXT (right?) and so searching for `perl.bat` should return that it is executable.